references/environment-variables.md

```diff
@@ -5,6 +5,7 @@
 - `MONGODB_URI`: MongoDB connection uri (required)
 - `VAULT_KEY`: secret key for the Kontena Vault (required)
 - `VAULT_IV`: initialization vector for the Kontena Vault (required)
+- `INTIIAL_ADMIN_CODE`: initial authorization code to obtain an access token for authenticating as the internal administrator (optional)
```

Comment:
typo fix needed, it's `INITIAL_ADMIN_CODE` :)

Reply:
`rqeuire 'kotnena'`